roles/openshift_node_group/tasks/bootstrap_config.yml

```diff
@@ -15,6 +15,9 @@
     line: "BOOTSTRAP_CONFIG_NAME={{ openshift_node_group_name }}"
     regexp: "^BOOTSTRAP_CONFIG_NAME=.*"
 
+- import_role:
```

Comment:
Not sure why is this required - why not leaving the call in `playbooks/openshift-node/private/configure_nodes.yml`?

Reply:
Ah, okay, makes sense